rasa/core/policies/form_policy.py

```diff
@@ -29,6 +29,17 @@ def __init__(
             featurizer=featurizer, priority=priority, max_history=2, lookup=lookup
         )
 
+    @staticmethod
+    def validate_against_domain(ensemble, domain):
```

Comment:
please add types

Reply:
Can't import `PolicyEnsemble` here as that'd cause a circular import